### PR TITLE
Update CVE-2020-13671.json

### PR DESCRIPTION
**Pull request by mlhess**

### CODE REVIEW

File: 2020/13xxx/CVE-2020-13671.json [hunk 1]
```diff
@@ -1,18 +1,87 @@
 {
```
Comment (cve-team) [inline 2020/13xxx/CVE-2020-13671.json:1]: {} is not allowed for {'vendor': {'vendor_data': [{'vendor_name': 'Drupal ', 'product': {'product_data': [{'product_name': 'Drupal Core', 'version': {'version_data': [{'version_name': '9.0', 'version_affected': '<', 'version_value': '9.0.8', 'platform': ''}, {'version_name': '8.9', 'version_affected': '<', 'version_value': '8.9.9', 'platform': ''}, {'version_name': '8.8', 'version_affected': '<', 'version_value': '8.8.11', 'platform': ''}, {'version_name': '7', 'version_affected': '<', 'version_value': '7.74', 'platform': ''}]}}]}}]}}

Comment (cve-team) [inline 2020/13xxx/CVE-2020-13671.json:1]: {} is not allowed for {'problemtype_data': [{'description': [{'lang': 'eng', 'value': 'Remote code execution'}]}]}

Comment (cve-team) [inline 2020/13xxx/CVE-2020-13671.json:1]: {} is not allowed for {'reference_data': [{'refsource': 'CONFIRM', 'url': 'https://www.drupal.org/sa-core-2020-012', 'name': ''}]}

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2020/13xxx/CVE-2020-13671.json (hunk 1 above)
Comment (cve-team) [inline 2020/13xxx/CVE-2020-13671.json:1]: A language tag was used multiple times: description.

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2020/13xxx/CVE-2020-13671.json [hunk 2]
```diff
@@ -1,18 +1,87 @@
...
+    ]
+  },
+  "description": {
+    "description_data": [
+      {
+        "lang": "eng",
+        "value": "Remote code execution vulnerability in Drupal Core allows an attacker to upload files leading to files being interpreted as the incorrect extension and served as the wrong MIME type or executed as PHP for certain hosting configurations."
+      },
+      {
+        "lang": "eng",
```
Comment (cve-team) [inline 2020/13xxx/CVE-2020-13671.json]: The system doesn't know how to handle two descriptions for the same language.  You will need to consolidate the descriptions.

File: 2020/13xxx/CVE-2020-13671.json [hunk 3]
```diff
@@ -1,18 +1,87 @@
...
+        "value": "This issue affects:\nDrupal  Drupal Core\n9.0 versions prior to 9.0.8;\n8.9 versions prior to 8.9.9;\n8.8 versions prior to 8.8.11;\n7 versions prior to 7.74."
+      }
+    ]
+  },
+  "references": {
+    "reference_data": [
+      {
+        "refsource": "CONFIRM",
+        "url": "https://www.drupal.org/sa-core-2020-012",
+        "name": ""
```
Comment (cve-team) [inline 2020/13xxx/CVE-2020-13671.json]: The name field for the reference cannot be blank if you are using a refsource.

**Comment by cristina479** [comment]
@mlhess Please resolve the auto validation errors so we can process your request. Thank you!

File: 2020/13xxx/CVE-2020-13671.json [hunk 4]
```diff
@@ -1,18 +1,83 @@
 {
```
Comment (cve-team) [inline 2020/13xxx/CVE-2020-13671.json:1]: MISC or CONFIRM, but name != url for reference.

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2020/13xxx/CVE-2020-13671.json (hunk 4 above)
Comment (cristina479) [reply 2020/13xxx/CVE-2020-13671.json:1]: I think removing the `name` field will resolve this issue.

File: 2020/13xxx/CVE-2020-13671.json [hunk 5]
```diff
@@ -1,18 +1,83 @@
...
+          }
+        ]
+      }
+    ]
+  },
+  "description": {
+    "description_data": [
+      {
+        "lang": "eng",
+        "value": "This issue affects:\nDrupal  Drupal Core\n9.0 versions prior to 9.0.8;\n8.9 versions prior to 8.9.9;\n8.8 versions prior to 8.8.11;\n7 versions prior to 7.74."
```
Comment (cristina479) [inline 2020/13xxx/CVE-2020-13671.json]: What is the issue? The description should have at a minimum a vulnerability (i.e., problem type) with sufficient detail as to demonstrate that the vulnerability is unique.

**Pull request merged**